multiqc/modules/rsem/rsem.py

```diff
@@ -0,0 +1,161 @@
+    """
+    Cutadapt module class, parses stderr logs.
+    Also understands logs saved by Trim Galore!
+    (which contain cutadapt logs)
+    """
+
+    def __init__(self):
+
+        # Initialise the parent object
+        super(MultiqcModule, self).__init__(name='Rsem', anchor='rsem',
+        href='https://deweylab.github.io/RSEM/',
+        info="RSEM (RNA-Seq by Expectation-Maximization) is a software package for"\
+             "estimating gene and isoform expression levels from RNA-Seq data.")
+    
+         # Find and load any featureCounts reports
+        self.rsem_mappable_data = dict()
+        self.rsem_mapped_data = dict()
+        self.rsem_keys  = list()
+        
+        for f in self.find_log_files('rsem'):
+            
+            if f['s_name'] in self.rsem_mapped_data:
+                log.debug("Duplicate sample name found! Overwriting: {}".format(f['s_name']))
+            else :
+                self.rsem_keys.append(f['s_name'])
+            self.rsem_mapped_data[f['s_name']] = self.parse_rsem_report(f)
+            
+            self.add_data_source(f)
+        # Filter to strip out ignored sample names
+        self.rsem_mappable_data = self.ignore_samples(self.rsem_mappable_data)
+        self.rsem_mapped_data = self.ignore_samples(self.rsem_mapped_data)
+
+        if len(self.rsem_mapped_data) == 0  :
+            log.info("Could not find any reports in {}".format(config.analysis_dir))
+            raise UserWarning
+
+        log.info("Found {} reports".format(len(self.rsem_mapped_data)))
+
+        # Write parsed report data to a file
+        self.write_data_file(self.rsem_mapped_data, 'multiqc_mappable_data')
+
+        # Basic Stats Table
+        self.rsem_stats_table()
+
+        # Assignment bar plot
+        self.add_section( plot = self.rsem_chart() )
+
+
+    def parse_rsem_report (self, f):
+        """ Parse the rsem cnt stat file. """
+        data = dict()
+        for l in f['f'].splitlines():
+            s = l.split(" ")
+            if len(s) > 3:
+                #Line N0 N1 N2 N_tot   
+                # N0, number of unalignable reads;
+                # N1, number of alignable reads; 
+                # N2, number of filtered reads due to too many alignments; N_tot = N0 + N1 + N2
+                data['Unalignable'] = s[0]
+                data['Alignable'] = s[1]
+                data['Filtered'] = s[2]
+                pass
+            elif len(s) == 3:
+                data['Unique'] = s[0]
+                data['Multi'] = s[1]
+                data['Uncertain'] = s[2]
+            else:
+                break
+        return data
+    
+    def rsem_stats_table(self):
+        """ Take the parsed stats from the rsem report and add them to the
+        basic stats table at the top of the report """
+
+        headers = OrderedDict()
+
+        headers['Unalignable'] = {
+            'title': '{} reads unalignable '.format(config.read_count_prefix),
```

Comment:
It's good to have the column titles as short as possible to keep the columns narrow. Here I think we can drop the word `reads` from each `title`.